webserver/src/main.rs

```diff
@@ -0,0 +1,417 @@
+// TODO: streaming support + APIs.
+use bughouse_chess::*;
```

Comment:
The (unwritten, sorry about that) `use` order that I'm following is:
std
external crates
local crates

Reply:
Done.